app/kubemci/cmd/remove_clusters_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/kubemci/pkg/kubeutils"
+)
+
+// Test to verify validate.
+// This tests all flags except `--gcp-project`, which is tested by the next test.
+func TestValidateRemoveClustersArgs(t *testing.T) {
+	// validateRemoveClustersArgs should return an error with empty options.
+	options := removeClustersOptions{}
+	if err := validateRemoveClustersArgs(&options, []string{}); err == nil {
```

Comment:
This looks like it could be a table-style test. But I suppose that could come in a follow-up PR.

Reply:
ack

Reply:
ack